sources/packages/web/src/services/AuthService.ts

```diff
@@ -67,6 +74,16 @@ export class AuthService {
    */
   priorityRedirect?: LocationAsRelativeRaw = undefined;
 
+  /**
+   * Function that makes a request to the API to log that the user has closed the browser/tab.
+   * @param event event received by the browser.
+   */
+  logUserClosedBrowser = async function (event: Event) {
+    event.preventDefault();
```

Comment:
Is this added to prevent the browser default behavior of showing the confirmation dialog?

Reply:
Yes, usually `beforeUnload` is used to confirm if the user wants to leave the page.

Reply:
So, if we do not add the `event.preventDefault();` the user will see the modal?

Reply:
Actually we checked that the default behavior of `beforeUnload` is to close the browser without any confirmation. If the confirmation is required then `event. preventDefault()` can be called. As we didn't have any confirmation message I removed it from this function.

I also removed the `await` from the call to the request because with await the browser prioritizes closing the browser and cancels the request.

Reply:
Thanks @andrepestana-aot. Sounds good to me 👍